spark/common/src/test/scala/org/apache/sedona/sql/functionTestScala.scala

```diff
@@ -1821,6 +1821,18 @@ class functionTestScala
       3) shouldBe None
   }
 
+  it("Should pass ST_RemoveRepeatedPoints") {
```

Comment:
I think we'd include all the example cases in the postgis document here - https://postgis.net/docs/ST_RemoveRepeatedPoints.html